CHANGELOG.md

```diff
@@ -1,12 +1,14 @@
 # Changelog
 
-## 0.5.0
+## 0.6.0
 
-### Known Issues
+Changed CLI option name from `--downstream-http-tls-ca-auth-disabled` to `--downstream-http-tls-ca-auth-enabled` https://github.com/PegaSysEng/ethsigner/pull/230
```

Comment:
Is this meant to go under a heading? Changes?

Reply:
Given it's the only thing and it's not really one of our typical headings I think it makes sense to just have it as is.